UN-OCHA/unocha-site
# [UNO-783] Upgrade to CD 9.1, remove cd-dropdown override

Refs: UNO-783

This upgrades the CD to 9.1 and removes the `cd-dropdown` override (for the "View all donors" on the funding page) as the changes to the script were ported upstream.

## Tests

1. Checkout the branch, clear the cache, run composer install or recreate local image
2. Check that the main menu dropdowns still work
3. Check the the "view all donors" on the `/our-funding` page still works
